docs/changelog.md

```diff
@@ -1,3 +1,20 @@
+## 0.23.0 (Aug 23, 2021)
+
+### Added
+
+* Add method for uploading files to chat.
+* Add method for downloading files from chat.
+
+### Changed
+
+* Add `data` and `files` fields to `HTTPRequest` for sending multipart/form-data in request.
+* Add `expected_type` field to `HTTPRequest` and `expected_type` property to `BaseBotXMethod`
+  to control the construction of the response when the non-json response of the request method.
```

Comment:
`to allow interacting with non JSON responses.`

Reply:
Done